script/bulk_import/vbulletin.rb

```diff
@@ -5,6 +5,44 @@
 class BulkImport::VBulletin < BulkImport::Base
 
   SUSPENDED_TILL ||= Date.new(3000, 1, 1)
+  CHARSET_MAP = {
+    "armscii8" => nil,
```

Comment:
Where did you get that list from? Do we need it? I mean, can't we just pass the string to the `encode` method? It's weird this isn't supported by the standard library...